src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -1154,6 +1177,29 @@ void CalcVRFCondenser(EnergyPlusData &state, int const VRFCond)
         } else {
             if (vrf.HeatEIRFPLR1 > 0) EIRFPLRModFac = CurveValue(state, vrf.HeatEIRFPLR1, max(vrf.MinPLR, HeatingPLR));
         }
+
+        if (EIRFPLRModFac < 0.0) {
+            if (vrf.HeatEIRFPLRErrorIndex == 0) {
+                ShowSevereMessage(state, fmt::format("{} \"{}\":", std::string(cVRFTypes(VRF_HeatPump)), vrf.Name));
+                ShowContinueError(state, fmt::format(" Heating EIR Modifier curve (function of PLR) output is negative ({:.3T}).", EIRFPLRModFac));
+                ShowContinueError(state,
+                                  fmt::format(" Negative value occurs using an outdoor air temperature of {:.1T} C and an average indoor air "
+                                              "wet-bulb temperature of {:.1T} C.",
+                                              CondInletTemp,
+                                              InletAirWetBulbC));
+                ShowContinueErrorTimeStamp(state, " Resetting curve output to zero and continuing simulation.");
```

Comment:
Cooling is OAT and IDWB, heating is indoor T and outdoor WB or DB, depending on outdoor temperature type. Check what the code does for heating to test the outdoor temperature type when evaluating performance curves.

    A19, \field Heating Performance Curve Outdoor Temperature Type
       \type choice
       \key DryBulbTemperature
       \key WetBulbTemperature

Reply:
It turned out here probably a more proper message should be given the PLR number. So changed this message text and variables according to that.

Reply:
That looks much better.